aide_design/floc_model.py

```diff
@@ -295,56 +291,61 @@ def gamma_humic_acid_to_coag(ConcAl, ConcNatOrgMat, NatOrgMat, coag):
 
 @u.wraps(None, [u.m, u.kg/u.m**3, u.kg/u.m**3, u.kg/u.m**3, None,
                 None, None, u.dimensionless], False)
-def _pacl_term(DiamTube, ConcClay, ConcAl, ConcNatOrgMat, NatOrgMat, 
+def pacl_term(DiamTube, ConcClay, ConcAl, ConcNatOrgMat, NatOrgMat,
                coag, material, RatioHeightDiameter):
+    """"""
```

Comment:
huh?

Reply:
Fraction of the surface area that is covered with coagulant that is not covered with humic acid.